cmd/operator-sdk/build/cmd.go

```diff
@@ -34,17 +34,19 @@ var (
 	imageBuildArgs string
 	imageBuilder   string
 	goBuildArgs    string
+	skipImage      bool
 )
 
 func NewCmd() *cobra.Command {
 	buildCmd := &cobra.Command{
-		Use:   "build <image>",
+		Use:   "build [<image>]",
 		Short: "Compiles code and builds artifacts",
 		Long: `The operator-sdk build command compiles the Operator code into an executable binary
 and generates the Dockerfile manifest.
 
 <image> is the container image to be built, e.g. "quay.io/example/operator:v0.0.1".
-This image will be automatically set in the deployment manifests.
+This image will be automatically set in the deployment manifests. <image> is optional
```

Comment:
WDYT about something as:
```suggestion
By default, this image will be automatically set in the deployment manifests. Note that you can use the flag --skip-image in the case you are looking for to only build and prepare the Dockerfile.
```

Reply:
Done, with a small change to wording.